src/test/colorconfig_test.cpp

```diff
@@ -0,0 +1,79 @@
+#include <gtest/gtest.h>
+#include <QColor>
+
+#include "test/mixxxtest.h"
+
+#include <preferences/hotcuecolorpalettesettings.h>
+#include <util/color/hotcuecolorpalette.h>
+
+class ColorConfigTest : public MixxxTest {};
+
+TEST_F(ColorConfigTest, TestSavingColorWithoutAlpha) {
```

Comment:
"TestSavingColorWithoutAlpha" is redundant in the name of a test. Simply "SaveColorWithoutAlpha" works.